docs/src/reference/remote_inputs.rst

```diff
@@ -24,12 +24,12 @@ All available genomes and metadata
 
 Entire metadata & sequences data is uploaded from the ``ncov-ingest`` workflows for each of the ``gisaid`` and ``open`` sources:
 
--  ``metadata.tsv.gz``
--  ``sequences.fasta.xz``
--  ``nextclade.tsv.gz``
--  ``aligned.fasta.xz`` (Alignment via `Nextclade <https://docs.nextstrain.org/projects/nextclade/en/stable/user/output-files.html#aligned-nucleotide-sequences>`__. The default reference genome is `MN908947 <https://www.ncbi.nlm.nih.gov/nuccore/MN908947>`__ (Wuhan-Hu-1))
--  ``additional_info.tsv.gz`` (GISAID only)
--  ``flagged_metadata.txt.gz`` (GISAID only)
+-  ``metadata.tsv.gz`` / ``metadata.tsv.zst``
+-  ``sequences.fasta.xz`` / ``sequences.fasta.zst``
+-  ``nextclade.tsv.gz`` / ``nextclade.tsv.zst``
+-  ``aligned.fasta.xz`` / ``aligned.fasta.zst`` (Alignment via `Nextclade <https://docs.nextstrain.org/projects/nextclade/en/stable/user/output-files.html#aligned-nucleotide-sequences>`__. The default reference genome is `MN908947 <https://www.ncbi.nlm.nih.gov/nuccore/MN908947>`__ (Wuhan-Hu-1))
+-  ``additional_info.tsv.gz`` / ``additional_info.tsv.zst`` (GISAID only)
+-  ``flagged_metadata.txt.gz`` / ``flagged_metadata.txt.zst`` (GISAID only)
```

Comment:
Maybe use ` and ` or `, ` for the separator instead of ` / ` so it's not confusable with a path? I had to do a double take.

![image](https://user-images.githubusercontent.com/79913/197898180-67435798-8e44-42fb-a1ec-588c2ca9b714.png)

Reply:
Ah, didn't think of `/` as a path separator, just a shorthand for `or`. I'll update!

Reply:
Updated in [force-push](https://github.com/nextstrain/ncov/compare/bf6bf933ef70645094d8a08cc3197739583a1c37..df80a63c4d46effaf182bbf4b55befc917a1c298).

Reply:
grazie!